swarm/tools/flow_studio/routes/health.py

```
@@ -55,3 +55,37 @@ async def api_health(state: FlowStudioState = Depends(get_state)) -> dict[str, A
             "validation": state.validation_data is not None,
         },
     }
+
+
+@router.post("/api/reload", response_model=schema.ReloadResponse if schema else None)
+async def api_reload(state: FlowStudioState = Depends(get_state)) -> dict[str, Any]:
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)